.github/workflows/build.yml

```diff
@@ -13,7 +13,9 @@ jobs:
   build:
     name: Compile and Test
     runs-on: ubuntu-latest
-
+    permissions:
```

Comment:
This shouldn't be required.

Reply:
I tried testing it without the permissions: line just now and got the following error 
``` Invalid workflow file
You have an error in your yaml syntax on line 16
``` 
where it then got stuck and refused to finish the checks. So it seems like it might be necessary?